README.md

```diff
@@ -85,6 +106,17 @@ template_parameters for the above html would be:
 {'variable1': 'Value 1', 'variable2': 'Value 2'}
 ```
 
+### Email Configuration Shell
+
+An email configuration resource can be added to Cloudshell blueprints using the driver included in the email-config-shell folder. 
+
+To install the package: 
+```commandline
+cd shell-config-shell
+shellfoundry install
+```
+When adding this resource to the inventory, Cloudshell will validate the SMTP server configuration defined in its attributes.
```

Comment:
Can you also add a note about when to this and a code example?
Something like:
"This shell is useful to avoid hard coding the configurations and SMTP server authentication details when there is a need to send a custom email during setup/teardown (or any other orchestration flow). The information saved in the Email Configuration resource should be used to initialize the EmailConfig object in the automation. 
Example for simple implementation in setup script:"
[add code example for usage in setup script]

Reply:
Updated the example to use GetResourceDetails